### PR TITLE
Merge configuration changes from ACME to CIME

### PR DESCRIPTION
**Pull request by mfdeakin-sandia**
This just copies the latest configuration files from ACME to CIME to fix Skybridge configuration changes and meet the new xmllint requirements

Test suite: scripts_regression_tests
Test baseline: 
Test namelist changes: 
Test status: BFB

User interface changes?: N

Update gh-pages html (Y/N)?: N

Code review: @jedwards4b

### CODE REVIEW

File: config/acme/machines/config_machines.xml [hunk 1]
```diff
@@ -117,76 +118,61 @@
...
-      <command name="load">craype/2.5.5</command>
+      <command name="load">craype/2.5.12.3</command>
       <command name="load">craype-ivybridge</command>
       <command name="rm">pmi</command>
-      <command name="load">pmi/5.0.10-1.0000.11050.0.0.ari</command>
+      <command name="load">pmi/5.0.12</command>
     </modules>
 
     <modules mpilib="mpi-serial">
+      <command name="rm">cray-netcdf-hdf5parallel</command>
```
Comment (jedwards4b) [inline config/acme/machines/config_machines.xml]: these look wrong, loading parallel io libraries with a mpi-serial build.

File: config/acme/machines/config_machines.xml [hunk 2]
```diff
@@ -289,10 +275,14 @@
       </modules>
 
       <modules mpilib="mpi-serial">
+        <command name="rm">cray-netcdf-hdf5parallel</command>
```
Comment (jedwards4b) [inline config/acme/machines/config_machines.xml]: ditto

File: config/acme/machines/config_machines.xml [hunk 3]
```diff
@@ -423,7 +412,9 @@
       </modules>
 
       <modules mpilib="mpi-serial">
-	<command name="rm">cray-netcdf-hdf5parallel</command>
+        <command name="rm">cray-netcdf-hdf5parallel</command>
```
Comment (jedwards4b) [inline config/acme/machines/config_machines.xml]: again here.

File: config/acme/machines/config_machines.xml [hunk 4]
```diff
@@ -876,14 +843,80 @@
...
+      <command name="purge"/>
+      <command name="load">sems-env</command>
+      <command name="load">sems-git</command>
+      <command name="load">sems-python/2.7.9</command>
+      <command name="load">sems-cmake</command>
+      <command name="load">gnu/4.9.2</command>
+      <command name="load">sems-intel/16.0.2</command>
+      <command name="load" mpilib="!mpi-serial">sems-openmpi/1.10.5</command>
+      <command name="load">mkl/16.0</command>
+      <command name="load">sems-netcdf/4.4.1/exo_parallel</command>
```
Comment (jedwards4b) [inline config/acme/machines/config_machines.xml]: this may have the same problem, i'm not sure

File: config/acme/machines/config_machines.xml [hunk 5]
```diff
@@ -1900,58 +2093,56 @@
...
-
-      <environment_variables compiler="pgiacc" mpilib="!mpi-serial">
-	<env name="PNETCDFROOT">/opt/cray/parallel-netcdf/1.7.0/PGI/15.3</env>
-      </environment_variables>
+    <!-- Set if compiler and mpilib  -->
+    <environment_variables compiler="pgiacc">
+      <!-- NOTE(wjs, 2015-03-12) The following line is needed for bit-for-bit reproducibility -->
+      <env name="CRAY_CPU_TARGET">istanbul</env>
+      <env name="CRAY_CUDA_MPS">1</env>
+<env name="NETCDFROOT">/opt/cray/netcdf-hdf5parallel/4.4.1.1/PGI/15.3/</env>
```
Comment (jedwards4b) [inline config/acme/machines/config_machines.xml]: could also be a problem for mpiserial

File: config/acme/machines/config_machines.xml (hunk 1 above)
Comment (mfdeakin-sandia) [inline config/acme/machines/config_machines.xml]: Aren't these removing the parallel modules?

File: config/acme/machines/config_machines.xml (hunk 3 above)
Comment (jedwards4b) [inline config/acme/machines/config_machines.xml]: yes, you are correct, I misread them.

**Comment by mfdeakin-sandia** [comment]
Sorry; I screwed up and had only started running scripts_regression_tests when I submitted this; it actually fails the xml test

**Pull request merged**